github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -484,6 +484,55 @@ presubmits:
           hostPath:
             path: /dev/vfio/
             type: Directory
+
+  - name: pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6
+    skip_branches:
+    - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
```

Comment:
Just for the initial debugging, right?

Reply:
Yep, so people is able to run the job with /test foobar.